.cycloid.yml

```diff
@@ -1,15 +1,28 @@
 # Configuration of the CI pipeline
-pipeline:
+name: 'Amazon S3 Bucket'
+canonical: 'infra-amazon-s3'
+description: 'This stack will deploy a S3 bucket on AWS.'
```

Comment:
Should be `this infrastructure` instead of `this stack`?

Reply:
:+1: